photon-core/src/main/java/org/photonvision/common/networking/NetworkUtils.java

```diff
@@ -31,12 +31,12 @@
 public class NetworkUtils {
     private static final Logger logger = new Logger(NetworkUtils.class, LogGroup.General);
 
-    public static enum NMType {
+    public enum NMType {
         NMTYPE_ETHERNET("ethernet"),
```

Comment:
once again, why

Reply:
Static is redundant here because it is an inner class. Is there a memory thingy I'm not aware of?